tools/test-case-generators/format-request-map.json

```diff
@@ -943,6 +943,21 @@
               "user": 0,
               "group": 0
             }
+          ],
+          "repositories": [
+            {
+              "id": "webmin-noarch",
+              "name": "Webmin - noarch",
+              "baseurls": [
+                "https://download.webmin.com/download/yum"
+              ],
```

Comment:
I'm not sure about adding a repository we don't control (or can rely on) as part of our tests.  I guess it's fine since they're not used during the build, but maybe we can add one of our own (maybe something like `sap-rhui-azure` from our snapshots) or even a fake one (like example.org)?